enclave/core/CMakeLists.txt

```diff
@@ -302,8 +351,25 @@ enclave_enable_code_coverage(oecore)
 if (CODE_COVERAGE)
   # Disable the complier flag to the following sources to
   # avoid conflicts.
-  set_source_files_properties(once.c sgx/calls.c sgx/init.c
-                              PROPERTIES COMPILE_FLAGS -fno-profile-arcs)
+  set_source_files_properties(once.c sgx/init.c PROPERTIES COMPILE_FLAGS
+                                                           "-fno-profile-arcs")
+  # Adding "-fno-profile-arcs" for Code Coverage, in addition to existing flags
+  # Don't add -Wno-frame-address for clang-10 as it's not supported
+  if (CMAKE_C_COMPILER_VERSION VERSION_EQUAL 10)
+    set_source_files_properties(
```

Comment:
In certain places we error out if ver less than 11, but here we are supporting 11 and 10.

Reply:
This inconsistency is because this particular block of code is to set specific properties for Clang-10, as opposed to omitting unsupported versions.